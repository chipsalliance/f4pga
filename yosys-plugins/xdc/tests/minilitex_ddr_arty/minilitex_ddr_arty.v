../../../../third_party/minilitex_ddr_arty/minilitex_ddr_arty.v