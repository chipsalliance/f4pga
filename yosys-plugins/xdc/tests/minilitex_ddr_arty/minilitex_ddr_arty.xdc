 ## serial:0.tx
set_property LOC D10 [get_ports serial_tx]
set_property IOSTANDARD LVCMOS33 [get_ports serial_tx]
# ## serial:0.rx
set_property LOC A9 [get_ports serial_rx]
set_property IOSTANDARD LVCMOS33 [get_ports serial_rx]
# ## clk100:0
set_property LOC E3 [get_ports clk100]
set_property IOSTANDARD LVCMOS33 [get_ports clk100]
# ## cpu_reset:0
set_property LOC C2 [get_ports cpu_reset]
set_property IOSTANDARD LVCMOS33 [get_ports cpu_reset]
# ## ddram:0.a
set_property LOC R2 [get_ports {ddram_a[0]} ]
set_property SLEW FAST [get_ports {ddram_a[0]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[0]} ]
# ## ddram:0.a
set_property LOC M6 [get_ports {ddram_a[1]} ]
set_property SLEW FAST [get_ports {ddram_a[1]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[1]} ]
# ## ddram:0.a
set_property LOC N4 [get_ports {ddram_a[2]} ]
set_property SLEW FAST [get_ports {ddram_a[2]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[2]} ]
# ## ddram:0.a
set_property LOC T1 [get_ports {ddram_a[3]} ]
set_property SLEW FAST [get_ports {ddram_a[3]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[3]} ]
# ## ddram:0.a
set_property LOC N6 [get_ports {ddram_a[4]} ]
set_property SLEW FAST [get_ports {ddram_a[4]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[4]} ]
# ## ddram:0.a
set_property LOC R7 [get_ports {ddram_a[5]} ]
set_property SLEW FAST [get_ports {ddram_a[5]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[5]} ]
# ## ddram:0.a
set_property LOC V6 [get_ports {ddram_a[6]} ]
set_property SLEW FAST [get_ports {ddram_a[6]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[6]} ]
# ## ddram:0.a
set_property LOC U7 [get_ports {ddram_a[7]} ]
set_property SLEW FAST [get_ports {ddram_a[7]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[7]} ]
# ## ddram:0.a
set_property LOC R8 [get_ports {ddram_a[8]} ]
set_property SLEW FAST [get_ports {ddram_a[8]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[8]} ]
# ## ddram:0.a
set_property LOC V7 [get_ports {ddram_a[9]} ]
set_property SLEW FAST [get_ports {ddram_a[9]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[9]} ]
# ## ddram:0.a
set_property LOC R6 [get_ports {ddram_a[10]} ]
set_property SLEW FAST [get_ports {ddram_a[10]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[10]} ]
# ## ddram:0.a
set_property LOC U6 [get_ports {ddram_a[11]} ]
set_property SLEW FAST [get_ports {ddram_a[11]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[11]} ]
# ## ddram:0.a
set_property LOC T6 [get_ports {ddram_a[12]} ]
set_property SLEW FAST [get_ports {ddram_a[12]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[12]} ]
# ## ddram:0.a
set_property LOC T8 [get_ports {ddram_a[13]} ]
set_property SLEW FAST [get_ports {ddram_a[13]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_a[13]} ]
# ## ddram:0.ba
set_property LOC R1 [get_ports {ddram_ba[0]} ]
set_property SLEW FAST [get_ports {ddram_ba[0]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_ba[0]} ]
# ## ddram:0.ba
set_property LOC P4 [get_ports {ddram_ba[1]} ]
set_property SLEW FAST [get_ports {ddram_ba[1]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_ba[1]} ]
# ## ddram:0.ba
set_property LOC P2 [get_ports {ddram_ba[2]} ]
set_property SLEW FAST [get_ports {ddram_ba[2]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_ba[2]} ]
# ## ddram:0.ras_n
set_property LOC P3 [get_ports ddram_ras_n]
set_property SLEW FAST [get_ports ddram_ras_n]
set_property IOSTANDARD SSTL135 [get_ports ddram_ras_n]
# ## ddram:0.cas_n
set_property LOC M4 [get_ports ddram_cas_n]
set_property SLEW FAST [get_ports ddram_cas_n]
set_property IOSTANDARD SSTL135 [get_ports ddram_cas_n]
# ## ddram:0.we_n
set_property LOC P5 [get_ports ddram_we_n]
set_property SLEW FAST [get_ports ddram_we_n]
set_property IOSTANDARD SSTL135 [get_ports ddram_we_n]
# ## ddram:0.cs_n
set_property LOC U8 [get_ports ddram_cs_n]
set_property SLEW FAST [get_ports ddram_cs_n]
set_property IOSTANDARD SSTL135 [get_ports ddram_cs_n]
# ## ddram:0.dm
set_property LOC L1 [get_ports {ddram_dm[0]} ]
set_property SLEW FAST [get_ports {ddram_dm[0]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dm[0]} ]
# ## ddram:0.dm
set_property LOC U1 [get_ports {ddram_dm[1]} ]
set_property SLEW FAST [get_ports {ddram_dm[1]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dm[1]} ]
# ## ddram:0.dq
set_property LOC K5 [get_ports {ddram_dq[0]} ]
set_property SLEW FAST [get_ports {ddram_dq[0]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[0]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[0]} ]
# ## ddram:0.dq
set_property LOC L3 [get_ports {ddram_dq[1]} ]
set_property SLEW FAST [get_ports {ddram_dq[1]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[1]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[1]} ]
# ## ddram:0.dq
set_property LOC K3 [get_ports {ddram_dq[2]} ]
set_property SLEW FAST [get_ports {ddram_dq[2]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[2]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[2]} ]
# ## ddram:0.dq
set_property LOC L6 [get_ports {ddram_dq[3]} ]
set_property SLEW FAST [get_ports {ddram_dq[3]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[3]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[3]} ]
# ## ddram:0.dq
set_property LOC M3 [get_ports {ddram_dq[4]} ]
set_property SLEW FAST [get_ports {ddram_dq[4]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[4]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[4]} ]
# ## ddram:0.dq
set_property LOC M1 [get_ports {ddram_dq[5]} ]
set_property SLEW FAST [get_ports {ddram_dq[5]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[5]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[5]} ]
# ## ddram:0.dq
set_property LOC L4 [get_ports {ddram_dq[6]} ]
set_property SLEW FAST [get_ports {ddram_dq[6]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[6]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[6]} ]
# ## ddram:0.dq
set_property LOC M2 [get_ports {ddram_dq[7]} ]
set_property SLEW FAST [get_ports {ddram_dq[7]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[7]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[7]} ]
# ## ddram:0.dq
set_property LOC V4 [get_ports {ddram_dq[8]} ]
set_property SLEW FAST [get_ports {ddram_dq[8]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[8]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[8]} ]
# ## ddram:0.dq
set_property LOC T5 [get_ports {ddram_dq[9]} ]
set_property SLEW FAST [get_ports {ddram_dq[9]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[9]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[9]} ]
# ## ddram:0.dq
set_property LOC U4 [get_ports {ddram_dq[10]} ]
set_property SLEW FAST [get_ports {ddram_dq[10]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[10]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[10]} ]
# ## ddram:0.dq
set_property LOC V5 [get_ports {ddram_dq[11]} ]
set_property SLEW FAST [get_ports {ddram_dq[11]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[11]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[11]} ]
# ## ddram:0.dq
set_property LOC V1 [get_ports {ddram_dq[12]} ]
set_property SLEW FAST [get_ports {ddram_dq[12]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[12]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[12]} ]
# ## ddram:0.dq
set_property LOC T3 [get_ports {ddram_dq[13]} ]
set_property SLEW FAST [get_ports {ddram_dq[13]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[13]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[13]} ]
# ## ddram:0.dq
set_property LOC U3 [get_ports {ddram_dq[14]} ]
set_property SLEW FAST [get_ports {ddram_dq[14]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[14]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[14]} ]
# ## ddram:0.dq
set_property LOC R3 [get_ports {ddram_dq[15]} ]
set_property SLEW FAST [get_ports {ddram_dq[15]} ]
set_property IOSTANDARD SSTL135 [get_ports {ddram_dq[15]} ]
set_property IN_TERM UNTUNED_SPLIT_40 [get_ports {ddram_dq[15]} ]
# ## ddram:0.dqs_p
set_property LOC N2 [get_ports {ddram_dqs_p[0]} ]
set_property SLEW FAST [get_ports {ddram_dqs_p[0]} ]
set_property IOSTANDARD DIFF_SSTL135 [get_ports {ddram_dqs_p[0]} ]
# ## ddram:0.dqs_p
set_property LOC U2 [get_ports {ddram_dqs_p[1]} ]
set_property SLEW FAST [get_ports {ddram_dqs_p[1]} ]
set_property IOSTANDARD DIFF_SSTL135 [get_ports {ddram_dqs_p[1]} ]
# ## ddram:0.dqs_n
set_property LOC N1 [get_ports {ddram_dqs_n[0]} ]
set_property SLEW FAST [get_ports {ddram_dqs_n[0]} ]
set_property IOSTANDARD DIFF_SSTL135 [get_ports {ddram_dqs_n[0]} ]
# ## ddram:0.dqs_n
set_property LOC V2 [get_ports {ddram_dqs_n[1]} ]
set_property SLEW FAST [get_ports {ddram_dqs_n[1]} ]
set_property IOSTANDARD DIFF_SSTL135 [get_ports {ddram_dqs_n[1]} ]
# ## ddram:0.clk_p
set_property LOC U9 [get_ports ddram_clk_p]
set_property SLEW FAST [get_ports ddram_clk_p]
set_property IOSTANDARD DIFF_SSTL135 [get_ports ddram_clk_p]
# ## ddram:0.clk_n
set_property LOC V9 [get_ports ddram_clk_n]
set_property SLEW FAST [get_ports ddram_clk_n]
set_property IOSTANDARD DIFF_SSTL135 [get_ports ddram_clk_n]
# ## ddram:0.cke
set_property LOC N5 [get_ports ddram_cke]
set_property SLEW FAST [get_ports ddram_cke]
set_property IOSTANDARD SSTL135 [get_ports ddram_cke]
# ## ddram:0.odt
set_property LOC R5 [get_ports ddram_odt]
set_property SLEW FAST [get_ports ddram_odt]
set_property IOSTANDARD SSTL135 [get_ports ddram_odt]
# ## ddram:0.reset_n
set_property LOC K6 [get_ports ddram_reset_n]
set_property SLEW FAST [get_ports ddram_reset_n]
set_property IOSTANDARD SSTL135 [get_ports ddram_reset_n]

set_property INTERNAL_VREF 0.675 [get_iobanks 34]

#create_clock -name clk100 -period 10.0 [get_nets clk100]
#
#set_false_path -quiet -to [get_nets -quiet -filter {mr_ff == TRUE}]
#
#set_false_path -quiet -to [get_pins -quiet -filter {REF_PIN_NAME == PRE} -of [get_cells -quiet -filter {ars_ff1 == TRUE || ars_ff2 == TRUE}]} ]
#
#set_max_delay 2 -quiet -from [get_pins -quiet -filter {REF_PIN_NAME == Q} -of [get_cells -quiet -filter {ars_ff1 == TRUE}]} ] -to [get_pins -quiet -filter {REF_PIN_NAME == D} -of [get_cells -quiet -filter {ars_ff2 == TRUE}]} ]
