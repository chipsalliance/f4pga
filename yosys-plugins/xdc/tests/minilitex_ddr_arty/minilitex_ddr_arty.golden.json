{
  "$iopadmap$top.cpu_reset": {
    "IOSTANDARD": "LVCMOS33",
    "IO_LOC_PAIRS": "cpu_reset:C2"
  },
  "$iopadmap$top.serial_rx": {
    "IOSTANDARD": "LVCMOS33",
    "IO_LOC_PAIRS": "serial_rx:A9"
  },
  "$iopadmap$top.serial_tx": {
    "IOSTANDARD": "LVCMOS33",
    "IO_LOC_PAIRS": "serial_tx:D10"
  },
  "IOBUF": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[0]:K5",
    "SLEW": "FAST"
  },
  "IOBUF_1": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[1]:L3",
    "SLEW": "FAST"
  },
  "IOBUF_10": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[10]:U4",
    "SLEW": "FAST"
  },
  "IOBUF_11": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[11]:V5",
    "SLEW": "FAST"
  },
  "IOBUF_12": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[12]:V1",
    "SLEW": "FAST"
  },
  "IOBUF_13": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[13]:T3",
    "SLEW": "FAST"
  },
  "IOBUF_14": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[14]:U3",
    "SLEW": "FAST"
  },
  "IOBUF_15": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[15]:R3",
    "SLEW": "FAST"
  },
  "IOBUF_2": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[2]:K3",
    "SLEW": "FAST"
  },
  "IOBUF_3": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[3]:L6",
    "SLEW": "FAST"
  },
  "IOBUF_4": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[4]:M3",
    "SLEW": "FAST"
  },
  "IOBUF_5": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[5]:M1",
    "SLEW": "FAST"
  },
  "IOBUF_6": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[6]:L4",
    "SLEW": "FAST"
  },
  "IOBUF_7": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[7]:M2",
    "SLEW": "FAST"
  },
  "IOBUF_8": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[8]:V4",
    "SLEW": "FAST"
  },
  "IOBUF_9": {
    "IN_TERM": "UNTUNED_SPLIT_40",
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dq[9]:T5",
    "SLEW": "FAST"
  },
  "OBUFTDS": {
    "IOSTANDARD": "DIFF_SSTL135",
    "IO_LOC_PAIRS": "ddram_dqs_p[0]:N2,ddram_dqs_n[0]:N1",
    "SLEW": "FAST"
  },
  "OBUFTDS_1": {
    "IOSTANDARD": "DIFF_SSTL135",
    "IO_LOC_PAIRS": "ddram_dqs_p[1]:U2,ddram_dqs_n[1]:V2",
    "SLEW": "FAST"
  },
  "OBUFTDS_2": {
    "IOSTANDARD": "DIFF_SSTL135",
    "IO_LOC_PAIRS": "ddram_clk_p:U9,ddram_clk_n:V9",
    "SLEW": "FAST"
  },
  "clkbuf": {
    "IOSTANDARD": "LVCMOS33",
    "IO_LOC_PAIRS": "clk100:E3"
  },
  "obuf_a0": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[0]:R2",
    "SLEW": "FAST"
  },
  "obuf_a1": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[1]:M6",
    "SLEW": "FAST"
  },
  "obuf_a10": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[10]:R6",
    "SLEW": "FAST"
  },
  "obuf_a11": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[11]:U6",
    "SLEW": "FAST"
  },
  "obuf_a12": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[12]:T6",
    "SLEW": "FAST"
  },
  "obuf_a13": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[13]:T8",
    "SLEW": "FAST"
  },
  "obuf_a2": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[2]:N4",
    "SLEW": "FAST"
  },
  "obuf_a3": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[3]:T1",
    "SLEW": "FAST"
  },
  "obuf_a4": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[4]:N6",
    "SLEW": "FAST"
  },
  "obuf_a5": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[5]:R7",
    "SLEW": "FAST"
  },
  "obuf_a6": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[6]:V6",
    "SLEW": "FAST"
  },
  "obuf_a7": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[7]:U7",
    "SLEW": "FAST"
  },
  "obuf_a8": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[8]:R8",
    "SLEW": "FAST"
  },
  "obuf_a9": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_a[9]:V7",
    "SLEW": "FAST"
  },
  "obuf_ba0": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_ba[0]:R1",
    "SLEW": "FAST"
  },
  "obuf_ba1": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_ba[1]:P4",
    "SLEW": "FAST"
  },
  "obuf_ba2": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_ba[2]:P2",
    "SLEW": "FAST"
  },
  "obuf_cas": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_cas_n:M4",
    "SLEW": "FAST"
  },
  "obuf_cke": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_cke:N5",
    "SLEW": "FAST"
  },
  "obuf_cs": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_cs_n:U8",
    "SLEW": "FAST"
  },
  "obuf_dm0": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dm[0]:L1",
    "SLEW": "FAST"
  },
  "obuf_dm1": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_dm[1]:U1",
    "SLEW": "FAST"
  },
  "obuf_odt": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_odt:R5",
    "SLEW": "FAST"
  },
  "obuf_ras": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_ras_n:P3",
    "SLEW": "FAST"
  },
  "obuf_rst": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_reset_n:K6",
    "SLEW": "FAST"
  },
  "obuf_we": {
    "IOSTANDARD": "SSTL135",
    "IO_LOC_PAIRS": "ddram_we_n:P5",
    "SLEW": "FAST"
  }
}